packages/uiweb/src/lib/hooks/chat/useChatProfile.ts

```diff
@@ -1,32 +1,41 @@
-import * as PushAPI from '@pushprotocol/restapi';
```

Comment:
if this hook is for initialisation, then it should be names something else like useInitializePushUser()